.gitignore

```diff
@@ -13,3 +13,5 @@ Cargo.lock
 # Added by cargo
 
 /target
+
+.idea
```

Comment:
Please don't write your IDE setting file. You can use `~/.gitignore_global` instead.